README.md

```diff
@@ -88,6 +88,11 @@ The check method has the following API:
   // Timing: sends a timing command with the specified milliseconds
   client.timing('response_time', 42);
 
+  // Timer: Records how long a given function takes to execute (in milliseconds),
+  // and then sends that value using 'client.timing'
```

Comment:
Can you make it more explicit that timer returns a function, and the parameters?  How about: 

// Timer: Returns a function that you call to record how long the first parameter takes to 
// execute (in milliseconds) and then sends that value using 'client.timing'.  The parameters
// after the first one match those in timer.

Reply:
Sounds good, will update 👍